### PR TITLE
Allow download of scanned images

### PR DESCRIPTION
**Pull request by jessieay**
With scanned image:

screen shot 2016-01-26 at 2 27 52 pm

Without scanned image:

screen shot 2016-01-26 at 2 27 40 pm

### CODE REVIEW

File: app/helpers/manifest_helper.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module ManifestHelper
+  def file_upload_button(manifest)
+    if manifest.content['uploaded_file'].present?
+      link_to 'Download File Upload', manifest_manifest_upload_path(manifest_id: @manifest.uuid), class: 'usa-button usa-button-outline'
```
Comment (jessieay) [inline app/helpers/manifest_helper.rb]: perhaps this text should be different?

Comment (jessieay) [inline app/helpers/manifest_helper.rb]: @zacharycohn

**Comment by jessieay** [comment]
@zacharycohn copy updated at 5c9f58e78af4327590382ec9ba2531e988b1efdb

**Comment by zacharycohn** [comment]
:+1:

**Comment by pkarman** [comment]
@jessieay merge conflicts, alas

**Comment by jessieay** [comment]
@pkarman updated :)

**Pull request merged**